dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -19,7 +19,7 @@
 
 using namespace Microsoft::Resources;
 
-typedef struct
+typedef struct _MrmObjects
```

Comment:
>_MrmObjects [](http://example.com/codeflow?start=15&length=11)

Needed for `/permissive-`?

Reply:
Yep. The classic Windows style is `typedef struct _Foo { ... } Foo;`.

Better would be simply `struct Foo {...};` since it's just C++ code, not part of a Flat-C API for C consumption. But I was hesitant to make deep changes, and there's other fixes needed in here anyway so I tried to be as minimally invasive as I could and leave a deeper cleanup refactoring to the MRT team follow up